sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,300 @@
+| [`ADMIN SHOW TELEMETRY`](/sql-statements/sql-statement-admin-show-telemetry.md) | Shows telemetry data. |
+| [`ADMIN`](/sql-statements/sql-statement-admin.md) | Performs various administrative tasks. |
+| [`FLUSH TABLES`](/sql-statements/sql-statement-flush-tables.md) | Flushes a table. |
+| [`SET <variable>`](/sql-statements/sql-statement-set-variable.md) | Sets a variable. |
+| [`SET NAMES`](/sql-statements/sql-statement-set-names.md) | Set a character set and collation. |
+| [`SPLIT REGION`](/sql-statements/sql-statement-split-region.md) | Splits a region into smaller regions. |
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) | Cancels a DDL job. |
+| [`ADMIN CHECK [TABLE\|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) | Checks the integrity of a table or index. |
+| [`ADMIN CHECKSUM TABLE`](/sql-statements/sql-statement-admin-checksum-table.md) | Computes and checks the checksum of a table. |
+| [`ADMIN CLEANUP`](/sql-statements/sql-statement-admin-cleanup.md) | Cleans up residual files or resources. |
+| [`ADMIN PAUSE DDL`](/sql-statements/sql-statement-admin-pause-ddl.md) | Pauses DDL operations. |
+| [`ADMIN RESUME DDL`](/sql-statements/sql-statement-admin-resume-ddl.md) | Resumes DDL operations. |
+| [`ADMIN SHOW DDL [JOBS\|JOB QUERIES]`](/sql-statements/sql-statement-admin-show-ddl.md) | Shows DDL jobs or job queries. |
+| [`ADMIN`](/sql-statements/sql-statement-admin.md) | Performs various administrative tasks. |
+| [`FLUSH TABLES`](/sql-statements/sql-statement-flush-tables.md) | Flushes a table. |
+| [`SET <variable>`](/sql-statements/sql-statement-set-variable.md) | Sets a variable. |
+| [`SET NAMES`](/sql-statements/sql-statement-set-names.md) | Set a character set and collation. |
+| [`SPLIT REGION`](/sql-statements/sql-statement-split-region.md) | Splits a region into smaller regions. |
+
+</CustomContent>
+
+## Data import and export
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`CANCEL IMPORT JOB`](/sql-statements/sql-statement-cancel-import-job.md) | Cancels an ongoing import job. |
+| [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) | Imports data into a table from an external source. |
+| [`LOAD DATA`](/sql-statements/sql-statement-load-data.md) | Loads data into a table from a file. |
+| [`SHOW IMPORT JOB`](/sql-statements/sql-statement-show-import-job.md) | Shows the status of an import job. |
+
+## Backup & restore
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ADMIN RECOVER INDEX`](/sql-statements/sql-statement-admin-recover.md) | Recovers a damaged index. |
+| [`BACKUP`](/sql-statements/sql-statement-backup.md) | Creates a backup of the database. |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) | Restores the cluster to a previous state. |
```

Comment:
```suggestion
| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) | Restores the cluster to a specific point in time. |
```

Reply:
I think naming it point in time might confuse people. Maybe "...to a specific snapshot"?

Reply:
```suggestion
| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) | Restores the cluster to a specific snapshot. |
```